src/couch_replicator_manager.erl

```diff
@@ -89,7 +89,6 @@ replication_started(#rep{id = {BaseId, _} = RepId}) ->
     #rep_state{rep = #rep{db_name = DbName, doc_id = DocId}} ->
         update_rep_doc(DbName, DocId, [
             {<<"_replication_state">>, <<"triggered">>},
-            {<<"_replication_state_reason">>, undefined},
```

Comment:
They should see triggered. Undefined there meant that the state reason was deleted if document was set to triggered state.